### PR TITLE
Adding Markdown link syntax for URL

### PR DESCRIPTION
**Pull request by JasonCard**
URL without using Markdown link syntax can cause broken hyperlink for localized languages, so adding the syntax to resolve the issue.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by Youssef1313** [comment]
This PR and other PRs for the same issue conflicts with #18470. However, you solved the issue in a different way than in #18470.

**Comment by MSDN-WhiteKnight** [comment]
The preferred fix IMO is to provide descriptive link text whenever possible for SEO/prettyness. Or revisit the whole sentence if it's not possible. For example, `listed at https://www.microsoft.com on the "Trademarks" webpage` could become the link to the concrete trademarks page, probably https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general.aspx 

If the goal is to workaround loc issue ASAP we can go with either syntax, but it would be good to make proper fix in the long run.

**Comment by JasonCard** [comment]
Fix in [#18470](https://github.com/dotnet/docs/pull/18470) will also work for loc. No preference from loc side.

File: docs/architecture/modern-web-apps-azure/index.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ This book is provided "as-is" and expresses the author's views and opinions. The
 
 Some examples depicted herein are provided for illustration only and are fictitious. No real association or connection is intended or should be inferred.
 
-Microsoft and the trademarks listed at https://www.microsoft.com on the "Trademarks" webpage are trademarks of the Microsoft group of companies.
+Microsoft and the trademarks listed at [https://www.microsoft.com](https://www.microsoft.com) on the "Trademarks" webpage are trademarks of the Microsoft group of companies.
```
Comment (mairaw) [inline docs/architecture/modern-web-apps-azure/index.md:33]: The syntax for bare URLs is to actually use angle brackets. I think it's preferred vs. repeating the url.

```suggestion
Microsoft and the trademarks listed at <https://www.microsoft.com> on the "Trademarks" webpage are trademarks of the Microsoft group of companies.
```

**Comment by JasonCard** [comment]
Closing as same fix is in [#18470](https://github.com/dotnet/docs/pull/18470).

**Pull request closed without merging**